get-started/code-experiments/README.md

```diff
@@ -0,0 +1,260 @@
+## DVC Get Started for Experiments
+
+This repository contains a project that trains a classifier for [Fashion
+MNIST][fmnist] dataset in Tensorflow. 
+
+[fmnist]: https://github.com/zalandoresearch/fashion-mnist
+
+It is used as a showcase for [`dvc exp`][gsdvcexp] commands to manage large
+number of experiments. 
+
+[gsdvcexp]: https://dvc.org/doc/start/experiments
+
+After [installing DVC][installdvc] and cloning the repository, you can run:
+
+[installdvc]: https://dvc.org/doc/install
+
+```console
+python -m venv .venv
+. .venv/bin/activate
+python -m pip install -r requirements.txt
+```
+
+`params.yaml` defines many parameters to modify with `dvc exp run --set-param/-S`
+option. For example to set the number of convolutional units in `cnn` model, you
+can use:
```

Comment:
Here `cnn` refers to the value of `model.name` parameter, and incidentally `cnn` is a CNN model 😄 It may not be _the CNN model_ for the project, though. We may add additional CNN models, this one is too simple.

Reply:
I think @casperdcl's suggestion was mainly on syntax and is much easier to read. Keeping `cnn` as a quote is secondary.

Reply:
updated